rules/jsxExpressionSpacingRule.ts

```diff
@@ -28,7 +28,10 @@ class JsxExpressionSpacingWalker extends Lint.RuleWalker {
 				this.createFailure(
 					node.getStart() + 1,
 					1,
-					'jsx expression should have one space after opening \'{\''
+					'jsx expression should have one space after opening \'{\'',
+					new Lint.Fix('jsx-expression-spacing', [
+						this.appendText(node.getStart() + 1, ' ')
```

Comment:
I added some tests to cover this case, since it was missing from the linter currently.